AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -3399,5 +3399,30 @@ public boolean ankiGetCardMark() {
         public int ankiGetCardFlag() {
             return mCurrentCard.getUserFlag();
         }
+
+        @JavascriptInterface
+        public int ankiGetCardReview() {
```

Comment:
This one's up for discussion, I still don't feel it's /great/, but it feels more descriptive than `Review`
```suggestion
        public int ankiGetCardReps() {
```